content/id/docs/contribute/participate/_index.md

```diff
@@ -0,0 +1,116 @@
+- `@sig-docs-{language}-reviewers` merupakan pengulas (_reviewer_)
+
+Setiap tim dapat di referensikan dengan `@name` mereka dalam komentar GitHub untuk berkomunikasi dengan setiap orang dalam grup.
+
+Terkadang tim Prow dan GitHub tumpang tindih (_overlap_) tanpa kecocokan sama persis. Untuk penugasan masalah, _pull request_, dan untuk mendukung persetujuan PR,
+otomatisasi menggunakan informasi dari berkas `OWNERS`.
+
+
+### Berkas OWNERS dan bagian yang utama (_front-matter_)
+
+Proyek Kubernetes menggunakan perangkat otomatisasi yang disebut prow untuk melakukan otomatisasi
+yang terkait dengan masalah dan _pull request_ dalam GitHub.
+[Repository website Kubernetes](https://github.com/kubernetes/website) menggunakan
+dua buah [prow _plugin_](https://github.com/kubernetes/test-infra/tree/master/prow/plugins):
+
+- blunderbuss
+- approve
+
+Kedua _plugin_ menggunakan berkas
+[OWNERS](https://github.com/kubernetes/website/blob/master/OWNERS) dan
+[OWNERS_ALIASES](https://github.com/kubernetes/website/blob/master/OWNERS_ALIASES)
+dalam level teratas dari repository GitHub `kubernetes/website` untuk mengontrol
+bagaimana prow bekerja dalam repositori.
+
+Berkas OWNERS berisi daftar orang-orang yang menjadi pengulas dan pemberi persetujuan dalam SIG Docs.
+Berkas OWNERS juga bisa terdapat dalam subdirektori, dan dapat menimpa peranan karena 
+dapat bertindak sebagai pengulas atau pemberi persetujuan berkas dalam subdirektori itu dan
+apa saja yang ada dibawahnya. Untuk informasi lebih lanjut tentang berkas PEMILIK pada umumnya, lihatlah
+[OWNERS](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md).
+
+Selanjutnya, berkas _markdown_ individu dapat menyimpan daftar pengulas dan pemberi persetujuan 
+pada bagian yang utama, baik dengan menyimpan daftar nama pengguna individu GitHub atau grup GitHub.
+
+Kombinasi dari berkas OWNER dan bagian yang utama dalam berkas _markdown_ menentukan
+saran kepada pemilik PR yang didapat dari sistem otomatis tentang siapa yang akan meminta ulasan teknis
+dan ulasan editorial untuk PR mereka.
+
+## Cara menggabungkan pekerjaan
+
+Ketika _pull request_ digabungkan ke cabang (_branch_) yang digunakan untuk mempublikasikan konten, konten itu dipublikasikan di http://kubernetes.io. Untuk memastikan bahwa
+kualitas konten yang kita terbitkan bermutu tinggi, kita membatasi penggabungan _pull request_ bagi para pemberi persetujuan
+SIG Docs. Beginilah cara kerjanya.
+
+- Ketika _pull request_ memiliki label `lgtm` dan `approve`, tidak memiliki label `hold`,
+  dan lulus semua tes, _pull request_ akan digabungkan secara otomatis.
+- Anggota organisasi Kubernetes dan pemberi persetujuan SIG Docs dapat menambahkan komentar
+  untuk mencegah penggabungan otomatis dari _pull request_ yang diberikan (dengan menambahkan komentar `/hold`
+  atau menahan komentar `/lgtm`).
+- Setiap anggota Kubernetes dapat menambahkan label `lgtm` dengan menambahkan komen `lgtm`
```

Comment:
I learned that komen is formal word
https://kbbi.kemdikbud.go.id/entri/komen

Reply:
Nice info!